doc/content/en/docs/++version++/IDL Language/_index.md

```diff
@@ -66,14 +66,18 @@ A Maven plugin is also provided to compile .avdl files. To use it, add something
       <executions>
         <execution>
           <goals>
-            <goal>idl</goal>
+            <goal>idl2java</goal>
```

Comment:
I am not a user of the Java SDK but this looks like a breaking change.
Old applications that use `<goal>idl</goal>` will break ?! I see no goal `idl` in the list of available [goals](https://github.com/apache/avro/pull/2568/files#diff-32270f815b71de14206b8c2ff8a75f363f5b468c12862ddbaa0e8ab415c07361R77-R80).

Reply:
Good point.. I should have noticed that.

That said, there's also work underway (for the Java SDK) to parse IDL schemas using the same code as the standard JSON schemas.

Going forward with this PR, we should thus recreate the old goal. Preferably as near-empty subclass of the new mojo (logging a warning directing users to the new name). As the parameters have not changed, this will ensure backwards compatibility.